packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -1555,6 +1555,7 @@ void main() {
     Widget runTest() {
```

Comment:
Reviewers: I'd recommend first looking at the additions in `ink_paint_test_utils`, because they're used in several tests